app/routes/admin/route.tsx

```diff
@@ -1,15 +1,14 @@
+import "./styles.css"
+
 import { Space, Container } from "@mantine/core"
+import { cssBundleHref } from "@remix-run/css-bundle"
 import { LoaderFunction, json, LinksFunction } from "@remix-run/node"
 import { Outlet } from "@remix-run/react"
-import styles from "./styles.css"
```

Comment:
if you aren't going to use this, then we should delete the file.

Reply:
This is already removed in another PR https://github.com/pokt-foundation/portal-platform/pull/616